ios/Exponent/Kernel/DevSupport/EXKernelDevKeyCommands.m

```diff
@@ -68,6 +68,45 @@ - (NSString *)description
 @interface EXKernelDevKeyCommands ()
 
 @property (nonatomic, strong) NSMutableSet<EXKeyCommand *> *commands;
++ (void)handleKeyboardEvent:(UIEvent *)event;
+
+@end
+
+@interface UIEvent (UIPhysicalKeyboardEvent)
+
+@property (nonatomic) NSString *_modifiedInput;
+@property (nonatomic) NSString *_unmodifiedInput;
+@property (nonatomic) UIKeyModifierFlags _modifierFlags;
+@property (nonatomic) BOOL _isKeyDown;
+@property (nonatomic) long _keyCode;
+
+@end
+
+
+@implementation UIApplication (EXKeyCommands)
+
+- (void)handleKeyUIEventSwizzle:(UIEvent *)event
```

Comment:
I would rather prefix it with `EX` or something specific to us, so it is clear when it appears in stacktraces or somewhere